YIF.Core.Service/Concrete/Services/SuperAdminService.cs

```diff
@@ -509,6 +511,38 @@ public async Task<ResponseApiModel<DescriptionResponseApiModel>> DeleteSpecialty
         }
 
 
+        public async Task<ResponseApiModel<DescriptionResponseApiModel>> ModifyIoE(string userId, JsonPatchDocument<InstitutionOfEducationPostApiModel> institutionOfEducationPostApiModel)
+        {
+            var result = new ResponseApiModel<DescriptionResponseApiModel>();
+
+            InstitutionOfEducationPostApiModel request = new InstitutionOfEducationPostApiModel();
+            institutionOfEducationPostApiModel.ApplyTo(request);
+
+            var validator = new InstitutionOfEducationPostApiModelValidator();
+            var validResult = await validator.ValidateAsync(request);
+
+            string ioEId = (await _institutionOfEducationAdminRepository.GetByUserId(userId)).InstitutionOfEducationId;
+            var currentInstitutionOfEducationDTO = await _institutionOfEducationRepository.Get(ioEId);
+
+            var newInstitutionOfEducationDTO = _mapper.Map<JsonPatchDocument<InstitutionOfEducationDTO>>(institutionOfEducationPostApiModel);
+
+            newInstitutionOfEducationDTO.ApplyTo(currentInstitutionOfEducationDTO);
+
+            #region imageSaving
+            if (request.ImageApiModel != null)
+            {
+                var serverPath = _env.ContentRootPath;
+                var folderName = _configuration.GetValue<string>("ImagesPath");
+                var path = Path.Combine(serverPath, folderName);
+
+                var fileName = ConvertImageApiModelToPath.FromBase64ToImageFilePath(request.ImageApiModel.Photo, path);
+                currentInstitutionOfEducationDTO.ImagePath = fileName;
+            }
+            #endregion
+
+            await _institutionOfEducationRepository.Update(_mapper.Map<InstitutionOfEducation>(currentInstitutionOfEducationDTO));
+
```

Comment:
there is no triangular bracket

Reply:
there is also no return value in the "ModifyIoE" method